docs/source/_templates/layout.html

```diff
@@ -12,6 +12,21 @@
   }();
 </script>
 
+{% set image = 'https://user-images.githubusercontent.com/4307001/105420214-98052d00-5c05-11eb-86cc-1d1722099472.png' %}
```

Comment:
How is this being hosted?

Reply:
It's currently referencing the attached image [here](https://github.com/alteryx/compose/issues/200#issue-791539766). Do we want to save and reference it from somewhere else?

Reply:
let's host it on s3, similar to how we do for fearuretools